distribution/docs/src/main/resources/content/_securityServices/expansion-service.adoc

```diff
@@ -5,12 +5,14 @@
 :order: 02
 :summary: Defines rulesets to map metacard and user attributes to more complete sets of values.
 
+=== {title}
+
 The expansion service defines rulesets to map metacard attributes and user attributes to more complete sets of values.
 For example if a user has an attribute "alphabet" that contained the value "full", the expansion service can be configured to expand the "full" value out to ["a","b","c",...].
 
 ==== Configuring Expansion Service
 
-To use the expansion service, modify the following two files within the `${home_directory}/etc` directory:
+To use the expansion service, modify the following two files within the `${branding-lowercase}-<version>/etc/pdp` directory:
```

Comment:
❓ Why was `${home_directory}` updated to `${branding-lowercase}-<version>`? I thought it should be `${home_directory}`.

Reply:
I saw this somewhere else in the documentation. Perhaps @ricklarsen can weigh in on it.

Reply:
I think it should be switched to what you're saying

Reply:
It should be `${home_directory}` according to https://codice.atlassian.net/wiki/spaces/DDF/pages/342032397/Text+Replacements.

Reply:
Cool, it's changed. Thanks